policyengine_canada/parameters/gov/provinces/mb/benefits/mbcb/reduction/six_or_more_children.yaml

```diff
@@ -0,0 +1,28 @@
+description: Manitoba Child Benefit phase out rate for families with six children.
```

Comment:
Manitoba phases the child benefit out for families with six children at this rate.